dist/legacy-assets/js/main.js

```diff
@@ -12730,25 +12731,20 @@ function articleEditor(collectionId, data) {
     data.description.metaDescription = $(this).val();
   });
 
-  /* The checked attribute is a boolean attribute, which means the corresponding property is true if the attribute
-   is present at all—even if, for example, the attribute has no value or is set to empty string value or even "false" */
-  var checkBoxStatus = function () {
-    if (data.description.nationalStatistic === "false" || data.description.nationalStatistic === false) {
-      return false;
-    } else {
-      return true;
-    }
-  };
+  $("#natStat-checkbox").click(function () {
+      data.description.nationalStatistic = $("#natStat-checkbox").prop('checked') ? true : false;
+  });
 
-  $("#metadata-list input[type='checkbox']").prop('checked', checkBoxStatus).click(function () {
-    data.description.nationalStatistic = $("#metadata-list input[type='checkbox']").prop('checked') ? true : false;
+  $("#articleType-checkbox").click(function () {
+      data.isPrototypeArticle = $("#articleType-checkbox").prop('checked') ? true : false;
   });
 
   // Save
   var editNav = $('.edit-nav');
   editNav.off(); // remove any existing event handlers.
 
   editNav.on('click', '.btn-edit-save', function () {
+    console.log(data);
```

Comment:
Possibly want to remove this unless we think logging this information is useful in prod?